### DISCUSSION TITLE
Will outline support other type of files?

### DISCUSSION THREAD

**Discussion by saber-li** [outline/outline#7761] [General]
Will outline support other types of file?  for example, .ppt, .xls, etc.

**Comment by tommoor** [comment]
What do you mean by support?

**Reply by saber-li** [reply]
Thank you for your reply. I think besides markdown files, there are many other types of files, such as PowerPoint, Excel, etc. Is it possible to upload these types of files and provide a preview function?

**Reply by tommoor** [reply]
I don't believe there is an open-source compatible way of previewing these file types as they are all proprietary file formats.

**Reply by saber-li** [reply]
https://github.com/kekingcn/kkFileView.git, what about this project, we use this project for file preview, and it supports a wide range of document preview features.

**Reply by tommoor** [reply]
Well, It's a completely different technology stack and written in Chinese so I don't think anyone is realistically going to host that in addition to Outline.
